src/layers/core/arc-layer/arc-layer.js

```diff
@@ -70,23 +68,41 @@ export default class ArcLayer extends Layer {
 
   _createModel(gl) {
     let positions = [];
+    let indices = [];
     const NUM_SEGMENTS = 50;
     for (let i = 0; i < NUM_SEGMENTS; i++) {
-      positions = [...positions, i, i, i];
+      positions = positions.concat([i, -1, 0, i, 1, 0]);
+      if (i > 0) {
+        /*
+         *  (-1)-------------_,(1)
+         *    |          _,-"   |
+         *    o      _,-"       o
+         *    |  _,-"           |
+         *  (-2)"--------------(0)
+         */
+        indices = indices.concat([
+          i * 2 - 2, i * 2 - 1, i * 2 + 1,
+          i * 2 - 2, i * 2 + 1, i * 2 + 0
+        ]);
+      }
     }
 
     const shaders = assembleShaders(gl, this.getShaders());
 
-    return new Model({
+    const model = new Model({
       gl,
       vs: shaders.vs,
       fs: shaders.fs,
       geometry: new Geometry({
-        drawMode: GL.LINE_STRIP,
```

Comment:
We now rely on default `drawMode` here? Maybe we could still spell it out for clarity? 

Also, would `GL.TRIANGLE_STRIP` work? It might be faster than using indices and align more with `LineLayer`?